dcicutils/obfuscation_utils.py

```diff
@@ -75,7 +76,64 @@ def is_obfuscated(value: str) -> bool:
     return isinstance(value, str) and bool(OBFUSCATED_VALUE.match(value))
 
 
-def obfuscate_dict(dictionary: dict, inplace: bool = False, show: bool = False,
+# def obfuscate_dict(dictionary: dict, inplace: bool = False, show: bool = False,
```

Comment:
Did you say you wanted to rename to `obfuscate_json` ?

Reply:
The "compatible" part (and it's not quite functionally compatible, as you note for tuples, is the synonym definition at the bottom of the file. It's call-compatible.